lib/active_record/connection_adapters/percona_adapter.rb

```diff
@@ -91,8 +92,8 @@ def supports_migrations?
         true
       end
 
-      def new_column(field, default, cast_type, sql_type = nil, null = true, collation = '', extra = '')
-        Column.new(field, default, cast_type, sql_type, null, collation, strict_mode?, extra)
+      def new_column(field, default, type_metadata, null, table_name, default_function, collation, comment)
```

Comment:
would you mind disabling this cop here? take a look at https://github.com/bbatsov/rubocop/blob/master/manual/configuration.md#disabling-cops-within-source-code